### ISSUE TITLE
error[E0554]

### ISSUE THREAD

**Issue by makorne** [manpen/rip_shuffle#1]
Hi!
Thank you for your intereresting crate!
But there is the error:

```
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /home/den/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rip_shuffle-0.1.0/src/lib.rs:3:12
```
Cheers!

**Comment by manpen** [comment]
Hi makorne,

thank's for reaching out and your interest in the crate!

The crate relies on features that were only available in the `nightly` compiler channel at time of writing. Is it possible for you to use the [nightly channel](https://forge.rust-lang.org/infra/channel-layout.html)?

I will check whether it is possible to work around the required unstable features, but this will probably take some time.

Best,
   Manpen

**Comment by makorne** [comment]
> Is it possible for you to use the [nightly channel](https://forge.rust-lang.org/infra/channel-layout.html)?

No. Never. Stable is more stable :)

**Comment by manpen** [comment]
I just released version `0.2.0` which should now compile with a stable channel compiler; though the binary might be a little bit faster if unstable features are available (since this allows us to prefetch data)
